tests/Microsoft.DotNet.Docker.Tests/SdkImageTests.cs

```diff
@@ -79,10 +79,13 @@ public async void VerifyBlazorWasmScenario(ProductImageData imageData)
                 return;
             }
 
-            // wasm-tools workload is not supported on Alpine or ARM, fall back to the unoptimized test
-            bool useWasmTools = !isAlpine && !imageData.IsArm;
+            // `wasm-tools` workload does not work on ARM
+            // `wasm-tools` is also not supported on Alpine for .NET < 9 due to https://github.com/dotnet/sdk/issues/32327
+            int[] unsupportedVersionsForAlpine = [6, 7, 8];
+            bool isSupportedVersionForAlpine = unsupportedVersionsForAlpine.Contains(imageData.Version.Major);
```

Comment:
I think the logic needs to be reversed here.

```suggestion
            bool isSupportedVersionForAlpine = !unsupportedVersionsForAlpine.Contains(imageData.Version.Major);
```